source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,145 @@
+---
+owner_slack: "#govuk-developers"
+title: Architectural summary of GOV.UK
+section: Applications
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The following content is a summary of the architecture of GOV.UK.
+
+This content is a less detailed version of the [Architectural deep-dive of GOV.UK](/manual/architecture-deep-dive.html), and is for both developers and non-developers to refer to.
+
+The content summarises the architecture by stating what happens when a user:
+
+- visits a page on GOV.UK
+- publishes content to GOV.UK
+
+As this is for both a developer and non-developer audience, the content contains [definitions](#definitions) at the end.
+
+## A user visits a page on GOV.UK
+
+This content will go through 3 scenarios where a user:
+
+- visits the GOV.UK homepage
+- visits a GOV.UK webpage
+- searches GOV.UK using the search page
+
+### A user visits the GOV.UK homepage
+
+The user visits https://www.gov.uk or https://gov.uk to get to the GOV.UK hpmepage.
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+[Fastly](https://www.fastly.com/) is a cloud computing services provider and content delivery network (CDN).
+
+GOV.UK uses the Fastly CDN to handle the majority of requests to GOV.UK. This reduces the load on GOV.UK Origin by around 70%.
+
+GOV.UK Origin is a stack of computers in the AWS cloud, and its entry point is a load balancer. Origin routes requests to the machine running the application that can respond to the request.
+
+Fastly also provides edge nodes or servers that are closer to our users, particularly those outside the UK.
+
+If Fastly has the homepage cached, Fastly returns the homepage to the user.
+
+If Fastly does not have the homepage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+The request from Router is then proxied (or forwarded) to an application called Frontend.
+
+[Frontend](https://github.com/alphagov/frontend) is a public-facing app that serves the majority of documents on the /government part of GOV.UK. These government pages are fetched from the Content Store by the appropriate application. Frontend is a replacement for the public-facing parts of the Whitehall application.
```

Comment:
I'm not sure if all of the above is true, but it feels like unnecessary detail in any case. I'd simplify to:

```suggestion
[Frontend](https://github.com/alphagov/frontend) is one of several public-facing apps used to render pages on GOV.UK. The homepage is just one of the pages that Frontend is responsible for; its content is hard-coded in the application.
```

Note I've also removed reference to Whitehall's frontend here, as we'd rather pretend it doesn't exist! I've also removed reference to the Content Store as it's not really applicable here.

Reply:
This comment still stands.

Reply:
whoops